lib/process-issue-comment.js

```diff
@@ -0,0 +1,52 @@
+module.exports = processIssueComment;
+
+const parseComment = require("./parse-comment");
+const toSafeGitReferenceName = require("./to-safe-git-reference-name");
+const setupRepository = require("./setup-repository");
+const getConfig = require("./get-config");
+const addContributor = require("./add-contributor");
+
+async function processIssueComment({ context, commentReply }) {
+  const commentBody = context.payload.comment.body;
+  const { who, action, contributions } = parseComment(commentBody);
+
+  if (action !== "add") {
+    commentReply.reply(`I could not determine your intention.`);
+    commentReply.reply(
+      `Basic usage: @all-contributors please add @someone for code, doc and infra`
+    );
+    commentReply.reply(
+      `For other usages see the [documentation](https://allcontributors.org/docs/en/bot/usage)`
+    );
```

Comment:
Wouldn't merging all three `reply` calls in one (with newlines added of course) be better? I don't see any benefits in having those lines given that the result is only one comment will all this text.